templates/trick/show.html.twig

```diff
@@ -0,0 +1,103 @@
+{%  extends 'base.html.twig' %}
+
+{% block title %}
+   Figure {{ trick.name }}
+{% endblock %}
+
+{% block header %}
+    <div class="container-fluid header p-0">
+        {% for media in medias %}
+            {% if media.header == 1 %}<img src="{{ media.path }}" alt="Media de la figure{{ trick.name }}" class="img-fluid"/>{% endif %}
+        {% endfor %}
+        <div id="titleArea" class="col-6 offset-3 ">
+            <h1 id="title" class="col-12 text-center p-0 mb-0 font-weight-bold">{{ trick.name | upper}}</h1>
+            <div class="col-12 d-flex justify-content-center"><a href="#section" class="text-center"><i class="icon fa fa-arrow-down m-1" style="font-size: 40px;color: white;"></i></a></div>
+        </div>
+
+    </div>
+{% endblock %}
+
+{% block body%}
+    <div class="container-fluid bg-light" id="section">
+        <div class="row justify-content-center">
+            <div class="col-12 d-flex justify-content-center mb-3" style="font-size: 1.2em;background-color: rgba(70,80,90,0.2)">
+                <div>
+                    <a href="#"><i class="icon fa fa-pencil " ></i></a>
+                    <form method="post"  style="display:inline-block" onsubmit="return confirm('Etes vous sure de vouloir supprimer?')">
+                        <input type="hidden" name="_method" value="DELETE">
+                        <input type="hidden" name="_token" value="{{ csrf_token('delete') }}">
+                        <button type="submit" class="icon fa fa-trash-o " style="background: none;border:none; cursor: pointer"></button>
+                    </form>
+                </div>
+            </div>
+        </div>
+        <span id="view"></span>
+        <div class="col-12 d-flex justify-content-center d-block d-sm-none mb-2" style="bottom: 15vw;">
+            <a href="#galery" class="btn btn-outline-info" onclick="showGalery()"> Voir la Galery</a>
+        </div>
+        <div id="galery">
+            {% include 'trick/_picturesGalery.html.twig' %}
+
+            {% set showPlayer = 0  %}
+            {% for media in medias %}
```

Comment:
Pareil que plus haut  `{% for media in trick.medias %}`

Reply:
ok modification réalisée